### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by Sameony**
Typography of Mirinda UI
image
image
image
image

### CODE REVIEW

File: Styles/general.css [hunk 1]
```diff
@@ -1,32 +1,49 @@
...
+}
+
+h5 {
+  font-size: 0.9rem;
+  font-weight: 300;
+  font-family: "Montserrat", sans-serif;
 }
 
+h6 {
+  font-size: 12px;
```
Comment (nakulsharma15) [inline Styles/general.css]: Here, you can use rem for font-size as other font sizes are also declared with rem

**Review by nakulsharma15 — COMMENTED** [review]
Suggested a minor change, everything else seems good.

**Review by shubhamkumar648 — APPROVED** [review]
code looks clean you can merge

File: Styles/general.css (hunk 1 above)
Comment (Sameony) [reply Styles/general.css]: right, but for very small values, px is appreciated, that's why i used px for this particular font size.

Comment (Sameony) [reply Styles/general.css]: will update it, thanks

**Review by nakulsharma15 — APPROVED** [review]

**Pull request merged**